dev-php/fpdf/metadata.xml

```diff
@@ -0,0 +1,21 @@
+<?xml version="1.0" encoding="UTF-8"?>
+<!DOCTYPE pkgmetadata SYSTEM "http://www.gentoo.org/dtd/metadata.dtd">
+<pkgmetadata>
```

Comment:
I know these aren't required, but it could be good to add upstream, and yourself as a maintainer.

Reply:
Thanks, done.